entity-types/ext-connector/definition.yml

```diff
@@ -0,0 +1,26 @@
+domain: EXT
+type: CONNECTOR
+synthesis:
+  rules:
+  - identifier: service
+    name: service
+    conditions:
+    - attribute: account
+      present: true
+    - attribute: region
+      present: true
+    - attribute: application
+      value: "sherpa"
+
+  tags:
+    service:
+    account:
+    region:
+
+dashboardTemplates:
+  newRelic:
+    template: dashboard.json
+
+configuration:
+  entityExpirationTime: QUARTERLY
```

Comment:
any reason to do quarterly? users commonly complain on types that have this TTL defined as is too long.

Reply:
these are not services,  but rather short lived processes that come and go.  We maybe able to trim down the TTL,  but not sure at this point.  I am experimenting a bit,  trying to figure out how the entity will appear...act over time.. etc.

Reply:
In case you're experimenting I would recommend to lower this value (DAILY maybe?). This would enable the platform to expire more quickly entities that are just experiments and also stick more truthfully to the nature of the entities you're creating.